cmd/shell.go

```diff
@@ -85,6 +85,9 @@ type Config struct {
 	}
 
 	RA struct {
+		DNSResolver string
```

Comment:
I think it's time that the DNS settings go into "common", or into their own "dns" instead of multiple individual programs' configs.